### PR TITLE
[Icons]: Update icons from Figma

### PR DESCRIPTION
**Pull request by brave-support-admin**
> @brave/leo@0.0.1 update-icons
> node ./src/scripts/update-icons.js

Ignoring icon default-false.svg because:
- has a raster image element
Ignoring icon rainbow-color.svg because:
- has a raster image element
Ignoring icon bitcoin.com-color.svg because:
- has a raster image element
Ignoring icon daum-color.svg because:
- has a raster image element
Ignoring icon openllama-color.svg because:
- has a raster image element

### CODE REVIEW

File: icons/redermode-column-default.svg
Comment (boocmp) [inline icons/redermode-column-default.svg:1]: There is a typo here. Should be 'readermode-column-default.svg'

File: icons/redermode-column-wide.svg
Comment (boocmp) [inline icons/redermode-column-wide.svg:1]: Same typo here

File: icons/redermode-column-default.svg
Comment (fallaciousreasoning) [reply icons/redermode-column-default.svg:1]: cc @aguscruiz

File: icons/redermode-column-wide.svg
Comment (fallaciousreasoning) [reply icons/redermode-column-wide.svg:1]: cc @aguscruiz

File: icons/redermode-column-default.svg
Comment (aguscruiz) [reply icons/redermode-column-default.svg:1]: Whoops, fixed!

File: icons/redermode-column-wide.svg
Comment (aguscruiz) [reply icons/redermode-column-wide.svg:1]: Fixed

**Review by fallaciousreasoning — APPROVED** [review]
LGTM

**Pull request merged**